lib/full-site-editing/edit-site-export.php

```diff
@@ -5,6 +5,39 @@
  * @package gutenberg
  */
 
+/**
+ * Parses wp_template content and injects the current theme's
+ * stylesheet as a theme attribute into each wp_template_part
+ *
+ * @param string $template_content serialized wp_template content.
+ *
+ * @return string Updated wp_template content.
+ */
+function _remove_theme_attribute_from_content( $template_content ) {
```

Comment:
#### Note:

`_remove_theme_attribute_from_content` has a lot of similarities to `_inject_theme_attribute_in_content`. Will consider consolidating both methods tomorrow morning.

Reply:
Thinking about it more, I'm leaning towards leaving both methods as is. I like how explicit the code is, and I don't see DRYing out the code buying us much.